src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -412,8 +409,9 @@ export class SavedObjectsRepository {
         const actualResult = indexFound
           ? bulkGetResponse?.body.docs[opensearchRequestIndex]
           : undefined;
-        const docFound = indexFound && actualResult.found === true;
-        if (docFound && !this.rawDocExistsInNamespace(actualResult, namespace)) {
+        const docFound = indexFound && actualResult?.found === true;
+        // @ts-expect-error MultiGetHit._source is optional
```

Comment:
Do we need a follow-up?

Reply:
[issue](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/1360) is created